src/py/flwr/server/app.py

```diff
@@ -598,3 +618,13 @@ def _add_args_fleet_api(parser: argparse.ArgumentParser) -> None:
         help=f"Fleet API REST server address. Default:'{ADDRESS_FLEET_API_REST}'",
         default=ADDRESS_FLEET_API_REST,
     )
+    rest_group.add_argument(
+        "--ssl-certfile",
+        help="Fleet API REST SSL certificate file (as a path str). Default:None",
+        default=None,
+    )
+    rest_group.add_argument(
+        "--ssl-keyfile",
+        help="Fleet API REST SSL private key file (as a path str). Default:None",
+        default=None,
+    )
```

Comment:
Shouldn't we add these as general args and make them the same across gRPC and REST?

Reply:
@charlesbvll can you look into it?

Reply:
@tanertopal @charlesbvll we could even put those new args (and their usage in gRPC) into a new PR

Reply:
I will open the new PR as soon as this one is merged